src/app/project/ontology/default-data/default-properties.ts

```diff
@@ -68,39 +72,43 @@ export class DefaultProperties {
                 {
                     icon: 'arrow_drop_down_circle',
                     label: 'Dropdown',
+                    description: 'Dropdown menu with values from predefined list',
                     subPropOf: Constants.HasValue,
                     objectType: Constants.ListValue,
                     guiEle: Constants.SalsahGui + Constants.HashDelimiter + 'List',     // 'Pulldown'
                     group: 'List'
-                },
-                {
-                    icon: 'check_box',
-                    label: 'Checkboxes',
-                    subPropOf: Constants.HasValue,
-                    objectType: Constants.ListValue,
-                    guiEle: Constants.SalsahGui + Constants.HashDelimiter + 'Checkbox',
-                    group: 'List'
-                },
-                {
-                    icon: 'radio_button_checked',
-                    label: 'Multiple choice',
-                    subPropOf: Constants.HasValue,
-                    objectType: Constants.ListValue,
-                    guiEle: Constants.SalsahGui + Constants.HashDelimiter + 'Radio',
-                    group: 'List'
                 }
+                // {
```

Comment:
is this code still needed?

Reply:
I'm not sure if we have to reactivate it one day!?